qiskit/primitives/base/base_primitive.py

```diff
@@ -17,11 +17,13 @@
 from abc import ABC
 
 from qiskit.providers import Options
+from qiskit.utils.deprecation import deprecate_func
 
 
 class BasePrimitive(ABC):
     """Primitive abstract base class."""
 
+    @deprecate_func(since="1.2", additional_msg="Use BasePrimitiveV2 instead.")
```

Comment:
`BasePrimitiveV2` does not exist.
```suggestion
    @deprecate_func(since="1.2", additional_msg="Use BaseSamplerV2 or BaseEstimatorV2 instead.")
```